src/components/StatusCard/stories.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { text, select, boolean } from '@storybook/addon-knobs';
+import StandardStatusCard from './Components/Standard';
+import CompactStatusCard from './Components/Compact';
+import StatusCard from './index';
+
+
+export default { title: 'Components|Display/Status Card' };
+
+export const General = () => (
+
+    <StatusCard
+        header={text('Header', 'hello')}
```

Comment:
For stories, for consistency, declaring the "knobs" as their own variables and referencing them in your demo is preferred to declaring them inline.